gratipay/chomp/__init__.py

```diff
@@ -53,6 +53,7 @@ def fetch_catalog(self):
 
 
 def stringify(obj):
+    # XXX What if there is a `\t` in `v`?
```

Comment:
Hm. We should probably opt for different [separator](http://initd.org/psycopg/docs/cursor.html#cursor.copy_from) then?